Modules/posixmodule.c

```diff
@@ -5359,16 +5354,17 @@ os_posix_spawn_impl(PyObject *module, path_t *path, PyObject *argv,
     }
 
     if (file_actions != Py_None) {
-
         /* There is a bug in old versions of glibc that makes some of the
          * helper functions for manipulating file actions not copy the provided
          * buffers. The use of `temp_buffer` here is a workaround that keeps the
          * python objects that own the buffers alive until posix_spawn gets called.
+         * posix_spawn_file_actions_addopen copies the value of **path** except for
+         * some old * versions of * glibc (<2.20). The usage of temp_buffer is
```

Comment:
What all these stars mean?

Reply:
Sorry, that was an error after wrapping the lines. Fixed in 54de1e2

Reply:
I am rewriting a bit this paragraph.